docs/reference/api.yml

```diff
@@ -0,0 +1,3204 @@
+---
+openapi: 3.0.1
+info:
+  title: Open Food Facts Open API
+  description: As a developer, the Open Food Facts API allows you to get information and contribute to the products database. You can create great apps to help people make better food choices and also provide data to enhance the database.
+  termsOfService: https://openweathermap.org/terms
+  contact:
+    name: Open Food Facts
+    url: https://slack-ssl-openfoodfacts.herokuapp.com/
+    email: contact@openfoodfacts.org
+  license:
+    name: License (MIT, Apache 2.0, etc)
+    url: https://opendatacommons.org/licenses/odbl/summary/index.html
+  version: "2"
+servers:
+- url: https://world.openfoodfacts.org
+  description: prod
+paths:
+  /api/v2/product/{barcode}:
+    get:
+      tags:
+      - Read Requests
+      summary: Get for a specific product by barcode
+      parameters:
+      - name: barcode
+        in: path
+        description: The barcode of the product to be fetched
+        required: true
+        style: simple
+        explode: false
+        schema:
+          type: string
+          example: "04963406"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/inline_response_200'
+components:
+  schemas:
+    inline_response_200:
```

Comment:
we should call it "product"

This is the schemas of a product. We will reuse it a lot.

Reply:
in fact /api/v2/{barcode} returns a response that contains a product object:

{
code: "3270160503070",
product: {  ... },
status: 1,
status_verbose: "product found"
}

Can we have a schema that we can call get_product_response, that includes a schema for product ?
That way we can reuse product is search_products_response.